rotkehlchen/db/snapshots.py

```diff
@@ -202,3 +228,78 @@ def _export(
             return False, str(e)
 
         return True, ''
+
+    def import_snapshot(
+        self,
+        balances_snapshot_file: Path,
+        location_data_snapshot_file: Path,
+    ) -> Tuple[bool, str]:
+        main_currency = self.db.get_main_currency()
+        balances_list = self._csv_to_dict(balances_snapshot_file)
+        location_data_list = self._csv_to_dict(location_data_snapshot_file)
+        # check if the headers match the type stored in the db
+        if tuple(balances_list[0].keys()) != ('timestamp', 'category', 'asset_identifier', 'amount', f'{main_currency.symbol.lower()}_value'):  # noqa: E501
+            return False, 'csv file has invalid headers'
+        if tuple(location_data_list[0].keys()) != ('timestamp', 'location', f'{main_currency.symbol.lower()}_value'):  # noqa: E501
+            return False, 'csv file has invalid headers'
```

Comment:
This can be combined into a bit if check with an or clause since the body is the same.